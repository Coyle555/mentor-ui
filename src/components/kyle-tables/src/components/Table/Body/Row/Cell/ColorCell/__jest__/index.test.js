jest.mock('react-color', () => {
	return { SliderPicker: (props) => <div>{JSON.stringify(props)}</div> };
});

import React from 'react';
import { ColorCell } from '../index';
import renderer from 'react-test-renderer';

test('Default edit color picker render', () => {
	const tree = renderer.create(<ColorCell />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Color picker with a color passed in', () => {
	const tree = renderer.create(<ColorCell color="#000" />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Color picker in edit mode', () => {
	const tree = renderer.create(<ColorCell editMode={true} />).toJSON();

	expect(tree).toMatchSnapshot();
});
