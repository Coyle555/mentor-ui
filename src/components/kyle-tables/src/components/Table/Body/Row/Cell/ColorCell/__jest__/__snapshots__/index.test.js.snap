// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color picker with a color passed in 1`] = `
<div
  className="mui-table-color-cell"
  style={
    Object {
      "backgroundColor": "#000",
    }
  }
/>
`;

exports[`Default edit color picker render 1`] = `
<div
  className="mui-table-color-cell"
  style={
    Object {
      "backgroundColor": "#fff",
    }
  }
/>
`;
